### PR TITLE
Migrate session tests

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: test/session_test.dart [hunk 1]
```diff
@@ -0,0 +1,189 @@
+// Copyright (c) 2013, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
```
Comment (zanderso) [inline test/session_test.dart]: This needs to import and use the versions from package:http_io/http_io.dart

**Comment by kevmoo** [comment]
Ha! Yup!

**Comment by kevmoo** [comment]
@zanderso – PTAL

**Review by zanderso — APPROVED** [review]
lgtm

**Pull request merged**